src/config.h

```diff
@@ -42,6 +42,7 @@
 #define CRAFT_KEY_CHAT 't'
 #define CRAFT_KEY_COMMAND '/'
 #define CRAFT_KEY_SIGN '`'
+#define AUTO_CAMERA_ROTATE_MODE 'C'
```

Comment:
I would suggest adding this new keybind as a CRAFT key because the rest of the keybinds are CRAFT keys so if this keybind were instead called "CRAFT_KEY_AUTO_CAMERA_ROTATE" I think that would fit better since it would allow anyone who looks at our code to look at config.h and see that our keybinds are CRAFT keys.